parm/stage/atmosphere_warm_RT.yaml.j2

```
@@ -0,0 +1,51 @@
+atmosphere_warm_RT:
+    mkdir:
+        {% for mem in range(first_mem, last_mem + 1) %}
+        {% set COMOUT_ATMOS_RESTART_PREV_MEM = COMOUT_ATMOS_RESTART_PREV_MEM_list[mem] %}
+        - "{{ COMOUT_ATMOS_RESTART_PREV_MEM }}"
+        {% set COMOUT_ATMOS_ANALYSIS_MEM = COMOUT_ATMOS_ANALYSIS_MEM_list[mem] %}
+        - "{{ COMOUT_ATMOS_ANALYSIS_MEM }}"
+        {% endfor %} # mem loop
+    copy:
+        #select restart members files (80) specific to the cycle
```

Nitpick.  It's easier to read comments with a space after the `#`:

```suggestion
        # Select restart members files (80) specific to the cycle